src/renderer_d3d11.cpp

```diff
@@ -3367,7 +3367,7 @@ namespace bgfx { namespace d3d11
 					data = (const char*)m_uniforms[handle.idx];
 				}
 
-#define CASE_IMPLEMENT_UNIFORM(_uniform, _dxsuffix, _type) \
+#define CASE_IMPLEMENT_UNIFORM(_uniform) \
```

Comment:
I think this macro should just be removed and it be coded straight but thats my opinion

Reply:
You can do that, but fix all backends, not just D3D11.

Reply:
Done. WebGPU uses some other pattern. D3D9 and GL had more complex macros so I left them alone.